resources/views/locations/print.blade.php

```diff
@@ -126,7 +126,11 @@
     	@endphp
     	
     	@foreach ($assets as $asset)
-
+            @php
+                if($snipeSettings->show_archived_in_list != 1 && $asset->assetstatus->name === 'Archived'){
```

Comment:
We can't always assume the name will be "Archived" though. We'd want to look at the status label *type* here, I think.

Reply:
I use ` $asset->assetstatus->archived` instead, I think that is better.